packages/cli-lib/sandboxes.js

```diff
@@ -40,6 +41,18 @@ async function deleteSandbox(parentAccountId, sandboxAccountId) {
   };
 }
 
+async function getSandboxUsageLimits(parentAccountId, sandboxAccountId) {
+  let resp;
+
+  try {
+    resp = await _getSandboxUsageLimits(parentAccountId, sandboxAccountId);
+  } catch (err) {
+    throw err;
+  }
+
+  return resp.usage;
```

Comment:
Seems a little strange to return `resp.usage` even if `resp` is undefined. Would it be possible to return something else like `null` in that case?

Reply:
Good call! I'll update the fetch types response as well

Reply:
Pushed changes for this to the new PR: https://github.com/HubSpot/hubspot-cli/pull/833